src/ptvsd/_vendored/pydevd/tests_python/test_debugger_json.py

```diff
@@ -982,6 +983,33 @@ def test_evaluate(case_setup):
         writer.finished_ok = True
 
 
+def test_exception_details(case_setup):
+    with case_setup.test_file('_debugger_case_exceptions.py') as writer:
+        json_facade = JsonFacade(writer)
+
+        writer.write_set_protocol('http_json')
+        writer.write_add_exception_breakpoint_with_policy(
+            'IndexError',
+            notify_on_handled_exceptions=2,  # Notify only once
+            notify_on_unhandled_exceptions=0,
+            ignore_libraries=1
+        )
+
+        json_facade.write_make_initial_run()
+        hit = writer.wait_for_breakpoint_hit(REASON_CAUGHT_EXCEPTION)
```

Comment:
@fabioz I am having issues with getting this test to hit here. The `ptvsd` test seems to be running fine. But i am not sure why this one does not get hit. do you see anything obvious that i might be missing?

Reply:
Not sure... I applied your changes here and it seems to break for me (maybe you have outdated `.pyds` locally?).

Repo and results from running those tests:

https://github.com/fabioz/PyDev.Debugger/tree/karthiknadig-issue1222
https://travis-ci.org/fabioz/PyDev.Debugger/builds/510299196

-- note that I only did an additional fix to the test after the break was hit: https://github.com/fabioz/PyDev.Debugger/commit/5c4bb7ac0d1a1c58e09ad61d5b399e3d5a59f204